ProcessLib/Parameter/ConstantParameter.cpp

```diff
@@ -25,12 +25,20 @@ std::unique_ptr<ParameterBase> createConstantParameter(
     // then required tag 'values'.
     {
         //! \ogs_file_param{prj__parameters__parameter__Constant__value}
-        auto const value = config.getConfigParameterOptional<double>("value");
+        auto const value =
+            config.getConfigParameterOptional<std::vector<double>>("value");
 
         if (value)
         {
-            DBUG("Using value %g for constant parameter.", *value);
-            return std::make_unique<ConstantParameter<double>>(name, *value);
+            if (value->size() != 1)
+            {
+                OGS_FATAL(
+                    "Expected to read exactly one value, but %d were given.",
+                    value->size());
+            }
```

Comment:
Maybe I misunderstood something. But why is only a scalar allowed? IMHO a constant vector/matrix is a vector/matrix with constant entries.

Reply:
You are right. Multiple values are also possible but with \<value**s**\> tag. for the \<value\> tag only scalar is allowed.